src/libAtomVM/mailbox.h

```diff
@@ -82,4 +83,12 @@ term mailbox_peek(Context *c);
  */
 void mailbox_remove(Context *c);
 
+/**
+ * @brief Free memory associated with a mailbox message.
+ *
+ * @details Free memory associated with a mailbox message.
+ * @param m the message to free.
+ */
+void mailbox_free_message(Message *m);
```

Comment:
we are using something`_destroy` naming around, the idea is that some kind of actions are executed before calling `free`. What's is your opinion here?

Reply:
_destroy works for me.  Whatever you think is the most consistent.

Reply:
_destroy works for me. Whatever you think is the most consistent.

Reply:
Done.